### PR TITLE
#120: API-core-SpaceAccessibility from md to rst

### PR DESCRIPTION
**Pull request by antoinemeyer5**
### THIS PR:
* API/core/SpaceAccessibility transition from .md to .rst

before (.md) | after (.rst) | after 2 (.rst)
:----------------: | :-------------: | :-------------: 
before_1 | after_1
before_2 | after_2 | after2_2
before_3 | after_3

### CODE REVIEW

File: docs/source/API/core/SpaceAccessibility.rst [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+    
+    Kokkos::SpaceAccessibility<MSp1, MSp2>::deepcopy
+
+.. _KokkosDeepCopy: view/deep_copy.html
+
+.. |KokkosDeepCopy| replace:: ``Kokkos::deep_copy``
+
+A compile-time value convertible to ``bool`` guaranteed to be ``true`` if and only if it is valid within the Kokkos programming model to |KokkosDeepCopy|_ from any (otherwise valid) instance of |KokkosView|_ type ``V2`` (with ``std::is_same<V2::memory_space, MSp2>::value`` equal to ``true``) to any (otherwise valid and otherwise compatible) instance of a ``View`` type ``V1`` (with ``std::is_same<V1::memory_space, MSp1>::value`` equal to ``true``).  In other words, if ``v2`` is a valid instance of ``V2`` and ``v1`` is a valid instance of ``V1`` (with shape and other attributes otherwise compatible with ``v2``), the following expression will be well-defined and valid in the Kokkos programming model:
+
+.. cpp:function:: Kokkos::deep_copy(v1, v2);
```
Comment (fnrizzi) [inline docs/source/API/core/SpaceAccessibility.rst:65]: make it a code block please

**Review by fnrizzi — APPROVED** [review]

**Review by ajpowelsnl — APPROVED** [review]
The page itself looks fine, however, the content is not understandable.  Would it be possible to flag this page for (future) polishing after everything is converted to .rst?  I separate the conversion activities from the polishing.  Maybe you could put `FIXME: CONTENT POLISHING` at the top of such pages ?

**Comment by antoinemeyer5** [comment]
> 

Like this ?

image

If so, I pushed the code so the PR can be merge.

**Comment by PhilMiller** [comment]
Please don't visibly mark pages unless they're fundamentally wrong in some way. Just open an issue for follow up here.

**Comment by antoinemeyer5** [comment]
> Please don't visibly mark pages unless they're fundamentally wrong in some way. Just open an issue for follow up here.

Okay I do that and remove the message on the page.

I was also thinking that this was very odd. I misunderstood "Maybe you could put FIXME: CONTENT POLISHING at the top of such pages ?".

Sorry and thanks for your help!

**Comment by ajpowelsnl** [comment]
> > Please don't visibly mark pages unless they're fundamentally wrong in some way. Just open an issue for follow up here.
> 
> Okay I do that (here: #269) and remove the message on the page.
> 
> I was also thinking that this was very odd. I misunderstood "Maybe you could put FIXME: CONTENT POLISHING at the top of such pages ?".
> 
> Sorry and thanks for your help!

Sorry -- I should have specified "FIXME" as a comment, and not something that would be visible in the page.  But I'm happy to go with Phil's suggestion.

**Review by ajpowelsnl — APPROVED** [review]
Conversion itself is fine.  Content polishing can happen another day.

**Pull request merged**